### PR TITLE
Add NFR for DG

### PR DESCRIPTION
**Pull request by shaelynl**

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -293,9 +293,11 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
 1.  Should work on any _mainstream OS_ as long as it has Java `11` or above installed.
 2.  Should be able to hold up to 1000 persons without a noticeable sluggishness in performance for typical usage.
```
Comment (kouyk) [inline docs/DeveloperGuide.md:295]: Maybe we can include dogs and programs in here too?

File: docs/DeveloperGuide.md [hunk 2]
```diff
@@ -293,9 +293,11 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
...
-3.  A user with above average typing speed for regular English text (i.e. not code, not system admin commands) should be able to accomplish most of the tasks faster using commands than using the mouse.
-
-*{More to be added}*
+3.  Should be usable by a tech novice who is not familiar with CLI. 
+4.  Should respond within 2 seconds.
+5.  A user with above average typing speed for regular English text (i.e. not code, not system admin commands) should be able to accomplish most of the tasks faster using commands than using the mouse.
+6.  Simple interface that is easy to navigate.
+7.  Not required to handle finance-related bookkeeping.
```
Comment (kouyk) [inline docs/DeveloperGuide.md:300]: I feel that this line a bit weird? It does make sense though if we have the feature to take note if someone has paid for the upcoming program.

File: docs/DeveloperGuide.md (hunk 1 above)
Comment (shaelynl) [reply docs/DeveloperGuide.md:295]: okay i will add that!

File: docs/DeveloperGuide.md (hunk 2 above)
Comment (shaelynl) [reply docs/DeveloperGuide.md:300]: oh, so we will include finance-related stuff also? okay no prob i will remove number 7!

**Review by wei-yutong — APPROVED** [review]

**Pull request merged**